### ISSUE TITLE
Very simple way to improve tracking number of keys in-game

### ISSUE THREAD

**Issue by geekley** [TestRunnerSRL/OoT-Randomizer#1985]
When you hold A on items screen, the game currently shows, for each dungeon, the number of keys you have and, in parenthesis, the numbers of keys *ever collected* for that dungeon.
The number in parenthesis is only useful if you memorize the total number of keys each dungeon has (or use a separate tracker). With a very simple adjustment, you can improve that.

It should instead show the ***number of keys left to collect*** (i.e., total minus what you have ever collected) -- so, for example, if a dungeon has 5 total keys, then it starts at 0(5); after collecting 1 key it goes to 1(4); after using it, it goes to 0(4), and so on -- the goal is to make both numbers decrease to zero (instead of going up to some total that I have to know for each dungeon).

A very simple change that would make the in-game tracker more useful :)

Also, just to improve UI a little, you could show uncollected map/compass/boss-key as semi-transparent or gray where it CAN be collected, so it's different from the other dungeons that don't have them, which you don't have to worry about.

**Comment by fenhl** [comment]
A problem with this is that it spoils whether a dungeon is MQ, circumventing half of the “maps & compasses give info” setting.

**Comment by geekley** [comment]
True. You could either make it a setting (with this disclaimer) or apply this only when Vanilla/MQ status is not randomized for any dungeon.

**Comment by Darkunov** [comment]
I feel like a more elegant/clean way to implement this would be to always display collected keys in this overlay, but only display the remaining amount of keys once the vanilla/mq status of the dungeon is revealed. 


Pros : 
- One implementation fits all, no need for an optional setting.
- Good QoL for players who start with vanilla/MQ info
- Rewards game knowledge for players who may consider going into the dungeon without this info
- Doesn't negatively affect the settings or experience of players who always play with keysy or keys within dungeon.

**Comment by geekley** [comment]
I believe you mean something like `0(1/?)` becoming `0(1/5)` once you get the map, right? Where `5` is the total keys in this case. Agree, that seems like the best implementation. It also won't confuse players that are used to the old behavior.

**Comment by cjohnson57** [comment]
Imo this would be much more confusing than helpful to just swap the meaning of the number, but showing x/y would be an okay solution

Also, how many keys you have left for a dungeon doesn't really matter in rando since you don't need all keys to finish or even full clear most dungeons. Learning how many keys you need to do what in each dungeon is a part of learning rando. See my doc if you need help with that (for vanilla):

https://docs.google.com/document/d/1zK9sqLcmYmYADr701NbEmEMwJKq6dO1YaYPw--B_75I/edit?usp=sharing
